objectscale-manager/alerts/issues/OBJSC-SP-0001_RECOVERY_STARTED_issue.yaml

```
@@ -0,0 +1,10 @@
+- description: "Object Scale service procedure started Recovery Process"
```

```suggestion
- description: "ObjectScale service procedure started Recovery Process"
```
ObjectScale is one word no space.  thats the official product name.  please fix in all descriptions.